cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/components/imageCarousel.isml

```diff
@@ -16,7 +16,14 @@
                 <isset name="cldImageURLsList" value="${product.images.cloudinaryProductImages ? product.images.cloudinaryProductImages.imageURLs : pdict.cloudinary.images.imageURLs}" scope="page" />
                 <isloop items="${cldImageURLsList}" var="image" status="loopStatus">
                     <div class='carousel-item <isif condition="${loopStatus.index == 0}">active</isif>'>
-                        <img class="d-block img-fluid cld-responsive" data-src="${image.url}" alt="${image.alt}" itemprop="image" />
+                        <img class="d-block img-fluid ${image.isResponsive ? 'cld-responsive' : ''}"
+                            <isif condition="${image.isResponsive}">
+                                data-src="${image.url}"
+                            <iselse/>
+                                src="${image.url}"
```

Comment:
where is the 'srcset' and 'sizes' attributes?

Reply:
It's not there, from the start. If you really need we can add that.

Reply:
yes, it's mandatory